packages/flame/example/lib/main.dart

```diff
@@ -60,9 +60,9 @@ class MyGame extends BaseGame with DoubleTapDetector, TapDetector {
   }
 
   @override
-  void onTapUp(TapUpDetails details) {
+  void onTapUp(TapUpInfo event) {
     final touchArea = Rect.fromCenter(
-      center: details.localPosition,
+      center: event.position.toOffset(),
```

Comment:
```suggestion
      center: event.raw.localPosition,
```

Reply:
we should probably have a fromCenter that takes in vector2, that is true the issue here (not using raw)

Reply:
We actually have an extension for that! Just changed